Index: net/proxy/proxy_service.cc
===================================================================
--- net/proxy/proxy_service.cc	(revision 91424)
+++ net/proxy/proxy_service.cc	(working copy)
@@ -478,7 +478,7 @@
   DCHECK(proxy_config_service);
 
   if (!ProxyResolverFactoryForSystem::IsSupported()) {
-    LOG(WARNING) << "PAC support disabled because there is no "
+    DLOG(WARNING) << "PAC support disabled because there is no "
                     "system implementation";
     return CreateWithoutProxyResolver(proxy_config_service, net_log);
   }
Index: net/ocsp/nss_ocsp.cc
===================================================================
--- net/ocsp/nss_ocsp.cc	(revision 91424)
+++ net/ocsp/nss_ocsp.cc	(working copy)
@@ -578,7 +578,7 @@
   net::URLRequestContext* request_context = g_request_context;
   pthread_mutex_unlock(&g_request_context_lock);
   if (request_context == NULL) {
-    LOG(ERROR) << "No URLRequestContext for OCSP handler.";
+    DLOG(ERROR) << "No URLRequestContext for OCSP handler.";
     // The application failed to call SetURLRequestContextForOCSP, so we
     // can't create and use net::URLRequest.  PR_NOT_IMPLEMENTED_ERROR is not an
     // accurate error code for this error condition, but is close enough.

